### ISSUE TITLE
Use Spotbugs version from Gradle/Maven plugin

### ISSUE THREAD

**Issue by dmoebius** [JetBrains/spotbugs-intellij-plugin#80]
Instead of bundling a fixed version of Spotbugs, the Spotbugs IntelliJ plugin should be able to pickup the Spotbugs implementation from the current Gradle/Maven project. This should be made an option.

So, if the plugin finds an implementation of Spotbugs in the Gradle/Maven build classpath, it uses this, or it has an option to fall back to the bundled version.

There should be some checks that ensure that the version on build classpath is not too old (in this case fall back to the bundled version).

This would probably solve issue #19, too.

**Comment by Cobollatin** [comment]
I agree, this extension is currently useless for me. I want the IDE to show me the bugs found by my Maven config.
